src/View/SSViewer.php

```diff
@@ -112,7 +112,7 @@ class SSViewer implements Flushable
      * @config
      * @var bool
      */
-    private static $rewrite_hash_links = true;
+    private static $rewrite_hash_links = false;
```

Comment:
Ya it's just straight broken and it's not an easy fix it would require probably very very large changes to SSViewer to correct the issue. The middleware replaces the functionality in SSViewer so disabling it here prevents overlap and fixes the issue referenced.

Reply:
Ya I think that's a good idea. I thought I seen someway in the docs with I think it was the password expired middleware that allowed you to remove a single middlewhere which is probably why I didn't add a config for that.